tools/plot/fio2gnuplot

```diff
@@ -198,7 +198,7 @@ def compute_temp_file(fio_data_file,disk_perf,gnuplot_output_dir, min_time, max_
 			# Index will be used to remember what file was featuring what value
 			index=index+1
 
-			time, perf, x, block_size = line[1]
+			time, perf, x, block_size = line[1][:4]
```

Comment:
Another option was to do time, perf, x, block_size,_

Reply:
@ErwanAliasr1 that will fail if any additional columns get added, which is why I went with this option.

Reply:
fair enough ;)